common/debug.c

```diff
@@ -0,0 +1,280 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
```

Comment:
Add `Copyright (C) 2020 BiiLabs...` as well since we modified the original source.